packages/ds-medicare-gov/docs/src/pages/components/NavigationMenu/NavigationMenu.docs.scss

```diff
@@ -1,6 +1,16 @@
 /*
```

Comment:
Can delete these lines too?

Reply:
I thought I needed these comments to format the page?  - `/*`

Reply:
Oh, I was thinking this whole page was being removed, is that not the case? If the whole page is being removed wouldn't this whole file go?

Reply:
No not removing the page - adding the deprecation notice only

Reply:
Omg I'm sorry I'm color blind this morning, for some reason I saw the green as red.. :facepalm: